Makefile

```diff
@@ -78,7 +78,7 @@ checks: lint go-mod-tidy
 
 generate:
 	$(CONTROLLER_GEN) object:headerFile="build/boilerplate/boilerplate.go.txt" paths="./pkg/apis/..."
-	$(CONTROLLER_GEN) crd:crdVersions=v1beta1,trivialVersions=true \
+	$(CONTROLLER_GEN) crd:crdVersions=v1,trivialVersions=true \
```

Comment:
per @rojkov feedback let's make sure both `crdVersions` are in sync and configurable using the same variable